olive/passes/onnx/conversion.py

```diff
@@ -117,7 +117,25 @@ def _run_for_config(
             # only handle dict for now since we cannot get the name of the input from a list/tuple
             if isinstance(dummy_inputs, dict):
                 dummy_input_keys = set(dummy_inputs.keys())
+
+                # handle dummy inputs for hf model with past, which has past_key_values
+                # match input names in `past_key_values.(hidden_layer_num).(key|value)` pattern
+                try:
+                    from transformers.modeling_utils import PreTrainedModel
+
+                    if issubclass(type(pytorch_model), PreTrainedModel):
```

Comment:
not really a problem, but is there a reason you didn't do something like `isinstance(pytorch_model, PreTrainedModel)`?

Reply:
I believe that `isinstance(pytorch_model, PreTrainedModel)` also works. I wrote it this way with reference to [transformers/onnx/convert.py#L481](https://github.com/huggingface/transformers/blob/main/src/transformers/onnx/convert.py#L481)

Reply:
I see. Was just curious if there was a reason for doing this. Thanks!